src/tests/stream-tcp-list.c

```diff
@@ -545,15 +545,15 @@ static int StreamTcpReassembleTest01(void)
     OverlapBSDBefore(UINT_MAX - 10);
     OverlapBSDSame(UINT_MAX - 10);
     OverlapBSDAfter(UINT_MAX - 10);
-    return 1;
+    PASS;
 }
 
 
 /** \test Vista Policy
  */
 static int StreamTcpReassembleTest02(void)
 {
-    OverlapVISTA(0);
+   FAIL_IF(OverlapVISTA(0) ==0);
```

Comment:
I think the calls in these macros are already covered by the FAIL/PASS API so we can skip these.